scripts/install_nlp_models.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
This script is no longer needed, it has been resolved with the latest 'source substorm.sh' script.